import java.util.Scanner;

public class Excep 
{
 public static void main(String args[]) 
 {
  int num1,
  num2, div; 
  Scanner sc= new Scanner (System.in);
  System.out.println("Enter value of num1:");
  num1 = sc.nextInt(); 
  System.out.println("Enter values of num2:");
  num2=sc.nextInt(); 
  try
  {
   div=num1/num2; 
   if(num2==0)
   {
    throw new ArithmeticException();
   }
   System.out.println(div);
  } 
  catch(ArithmeticException e)
  {
   System.out.println(e);
  } 
  System.out.println("The array is array[]=[4,5]");
  int array[]={4,5}; 
  try
  { 
   System.out.println(array[3]);
  }
  catch(Exception e)
  { 
   System.out.println("The error found is" +e);
  } 
 } 
}